mw-webapp/src/dataAccessLogic/WayPreviewDAL.ts

```diff
@@ -28,39 +71,49 @@ export class WayPreviewDAL {
   public static async getWaysPreviewByUuids(wayUuids: string[]): Promise<WayPreview[]> {
     const waysDTO = wayUuids.length !== 0 ? await WayService.getWaysDTOByUuids(wayUuids) : [];
```

Comment:
```suggestion
    const waysDTO = wayUuids.length ? await WayService.getWaysDTOByUuids(wayUuids) : [];
```

check in other places

Reply:
Removed everywhere